lib/src/widgets/chat/ds_quick_reply.widget.dart

```diff
@@ -0,0 +1,122 @@
+import 'dart:convert';
+
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+class DSQuickReply extends StatelessWidget {
+  final DSAlign align;
+  final Map<String, dynamic> content;
+  final Function? onSelected;
+  final bool hideOptions;
+
+  const DSQuickReply({
+    Key? key,
+    required this.align,
+    required this.content,
+    this.onSelected,
+    this.hideOptions = false,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: align == DSAlign.left
+          ? CrossAxisAlignment.start
+          : CrossAxisAlignment.end,
+      children: [
+        DSTextMessageBubble(
+          text: content['text'],
+          align: align,
+        ),
+        Align(
+          alignment: Alignment.bottomLeft,
+          child: _buildQuickReply(),
+        )
```

Comment:
I suggest using a comma at this point. depending on the editor configuration, the indentation may break.

![image](https://user-images.githubusercontent.com/753061/190657780-ff257b0d-21da-4243-9722-d1fc05298dfc.png)

Reply:
![image](https://user-images.githubusercontent.com/14128784/190690680-602fe476-e2af-4881-aff3-c5fcab7f4733.png)


here is ok, I don't think it's necessary to add the comma in these cases. we should check your configuration, to keep it the same for all developers.

what do you think?